microsite/data/plugins/ibm-apic-backend.yaml

```diff
@@ -0,0 +1,16 @@
+---
+title: IBM APIC Backend
+author: croz.net
+authorUrl: https://croz.net/?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=backstage-ibm-apic-backend
+category: API Management
+description: Bring IBM APIC to Backstage.
+documentation: https://croz.net/news/backstage-plugin-ibm-apic?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=backstage-ibm-apic-backend
```

Comment:
Ideally this should go to the repo `README` or a dedicated documentation page and not a blog post.

Reply:
I'm working we my team, will update the discussion when ready.